app/assets/javascripts/timeline.js

```diff
@@ -209,6 +224,17 @@ samson.controller("TimelineCtrl", function($scope, $window, $timeout, Deploys, S
   $scope.timelineDeploys = Deploys;
   $scope.deploys = Deploys.entries;
 
+
+  if (!$scope.selectedTimeFormat) {
+      $scope.selectedTimeFormat = document.getElementById("default_time_format").value;
+      $scope.timeFormat = $scope.selectedTimeFormat;
+  }
+
+
+  $scope.changeTime = function() {
+    $scope.selectedTimeFormat = $scope.timeFormat;
```

Comment:
ok, did some poking into this.  I think that ideally this is the kinda thing that would go into rootScope after app initialization.  this doesn't seem to really work that way though as we go through the config/init stuff on ever page change.  probably best to leave it here unless we want to re-factor the angular stuff more.